src/test/java/com/wikia/webdriver/common/contentpatterns/PageContent.java

```diff
@@ -141,6 +141,10 @@
   //Customized toolbar
   public static final String FOLLOW = "follow";
 
+  //Mobile preview
+  public static final String MOBILE_PREVIEW_VIDEO = "MobilePreviewVideoTest";
+  public static final String MOBILE_PREVIEW_IMAGE = "MobilePreviewImageTest";
```

Comment:
Those variables are used once and are not reusable, please move them to file where you are using them